src/NetAppFiles/NetAppFiles/help/New-AzNetAppFilesVolume.md

```diff
@@ -130,6 +133,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -MountTargets
```

Comment:
The name of this parameter should be in *singular* form.

Reply:
And I don't see any implementation of it in `NewNetAppFilesVolume.cs`. Did you forget to add it?

Reply:
In fact this help section was supposed to be removed already but for some reason was not merged. I've updated the file.